tests/dev/test_run.py

```diff
@@ -20,14 +20,11 @@ def runTest(self):
         result = self.runner.invoke(yoda.cli, ['run', 'tests/resources/test_code.py'])
         self.assertEqual(result.exit_code, 0)
         output_string = str(result.output.encode('ascii', 'ignore').decode('utf-8')).strip()
-        self.assertTrue("Compiling code.." in output_string)
 
         result = self.runner.invoke(yoda.cli, ['run', 'no_file.py'])
-        self.assertEqual(result.exit_code, 0)
+        self.assertEqual(result.exit_code, 1)
         output_string = str(result.output.encode('ascii', 'ignore').decode('utf-8')).strip()
-        self.assertTrue("Please re-check the PATH and try again." in output_string)
 
-        result = self.runner.invoke(yoda.cli, ['run', 'logo.png'])
+        result = self.runner.invoke(yoda.cli, ['run', '../logo.png'])
```

Comment:
The path should be `logo.png` instead of `../logo.png` as `../logo.png` will result in a wrong path. Hence reducing the coverage.